root/view/src/main/java/at/fhv/itb5c/view/login/LoginController.java

```diff
@@ -49,7 +52,10 @@ private void login() {
 			if (sessionID == null) {
 				ErrorPopUp.invalidLoginCredentials();
 			} else {
+				// valid credentials
 				AppState.getInstance().setSessionID(sessionID);
+				IUserRMI currentUser = RMIClient.getRMIClient().getApplicationFacade().getCurrentUser(sessionID);
```

Comment:
the current user is also saved in the appstate when the session id is saved into the app state so it is no nessecary to load it here